tcms_tenants/apps.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=import-outside-toplevel
+from django.apps import AppConfig as DjangoAppConfig
+from django.core.checks import register
+
+
+class AppConfig(DjangoAppConfig):
+    name = "tcms_tenants"
```

Comment:
Hmm. I can't find how this app currently gets into INSTALLED_APPS. I don't think it is listed there and it is surprising that it even loads and works. Need to investigate more.

Reply:
Found it https://github.com/kiwitcms/Kiwi/blob/master/tcms/settings/common.py#L293